sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -796,7 +796,7 @@ public async Task GetDataFeeds(bool useTokenCredential)
 
             var dataFeedCount = 0;
 
-            await foreach (DataFeed dataFeed in adminClient.GetDataFeedsAsync())
+            await foreach (DataFeed dataFeed in adminClient.GetDataFeedsAsync(new GetDataFeedsOptions()))
```

Comment:
Why does this need to change?  IIRC, the goal is for tests to pass without changing anything.

Reply:
This call was ambiguous between protocol and convenience method

![image](https://user-images.githubusercontent.com/45216704/161855356-7513569a-7aa1-483d-94e1-04d992c9f563.png)


Protocol Method 
```C#
public virtual AsyncPageable<BinaryData> GetDataFeedsAsync(string dataFeedName = null, string dataSourceType = null, string granularityName = null, string status = null, string creator = null, int? skip = null, int? maxpagesize = null, RequestContext context = null)
```

Convenience Method 
```C#
public virtual AsyncPageable<DataFeed> GetDataFeedsAsync(GetDataFeedsOptions options = default, CancellationToken cancellationToken = default)
```
To fix the the test, I passed `options`.

 We can rename the convenience method to `GetDataFeedsValuesAsync` as per the discussion [here](https://github.com/Azure/autorest.csharp/issues/2014#issuecomment-1059305328). I also added `Value` suffix in the [API view](https://apiview.dev/Assemblies/Review/a9812a1691e54ab4bf52540c4e54d433?diffRevisionId=9260bdbf10e346b9ba8dac7399ea885b&doc=False&diffOnly=False&revisionId=df651ea7349a46479efc1eb960d74246#Azure.AI.MetricsAdvisor.Administration.MetricsAdvisorAdministrationClient).

I didn't add Value suffix here because I thought we want to use the same method name as released HLC library.

Reply:
Yes, we should change the name of one of the methods, rather than changing the tests.  Since we are not changing the MA HLC API for this experiment, we should change the name of the protocol method.  @pshao25 has a PR for partner-facing guidance for this here: https://github.com/Azure/azure-sdk-for-net/pull/27559.  If it doesn't address this case, could you add feedback to the PR?

Reply:
@pshao25's [doc](https://github.com/Azure/azure-sdk-for-net/blob/4352e57d31bdf2271e2d55dfc4dc0212bcaf320e/doc/Data%20Plane%20Code%20Generation/DeveloperDrivenEvolution.md#rename-method-if-needed) covers this case.

Based on that I renamed DPG method `GetDataFeeds` to `GetDataFeedsValues`

Reply:
@pshao25's [doc](https://github.com/Azure/azure-sdk-for-net/blob/4352e57d31bdf2271e2d55dfc4dc0212bcaf320e/doc/Data%20Plane%20Code%20Generation/DeveloperDrivenEvolution.md#rename-method-if-needed) covers this case.

Based on that I renamed DPG method `GetDataFeeds` to `GetDataFeedsValues` and reverted the test case.

Reply:
Per offline conversation with @KrzysztofCwalina, let's use the suffix `Raw` instead of `Values` when we're changing the name of the protocol method and not the convenience API.  I'll add a comment to Pan's PR to add this as well.

Reply:
I've renamed listDataFeeds -> listDataFeedsRaw [here](https://github.com/Azure/azure-sdk-for-net/blob/f669ddb0d4f0965ac91cbdc5e2665ca27eea56dd/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/autorest.md#rename-dpg-method-name-listdatafeeds---listdatafeedsvalues). Since this is the paging method, generator is making it plural, so the method name is generated as [GetDataFeedsRaws](https://github.com/Azure/azure-sdk-for-net/blob/f669ddb0d4f0965ac91cbdc5e2665ca27eea56dd/sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated/MetricsAdvisorAdministrationClient.cs#L3053).

Reply:
Issue tracking here - https://github.com/Azure/azure-sdk-for-net/issues/28123

Reply:
Updated based on https://github.com/azure/azure-sdk-for-net/issues/28123#issuecomment-1106838980